Sources/Clappr/Classes/Base/Container.swift

```diff
@@ -118,6 +123,32 @@ open class Container: UIObject {
         }
     }
 
+    #if os(iOS)
```

Comment:
I tend to re-think about having the container handling such responsibility.

having platform checks like`#if os(iOS)` may suggest we want to move all this orientation thing to another class.

What do you think about having a `DeviceOrientationPlugin` responsible for listening to  `UIDevice.orientationDidChangeNotification` and triggering the device rotation events?

This way we can add this plugin to the iOS Player plugins list (since `tvOS` will not take advantage of it)

Reply:
Nice

=]